tests/test_halo.py

```diff
@@ -351,6 +351,47 @@ def test_spinner_interval_dict(self):
         spinner = Halo(spinner={'interval': 321, 'frames': ['+', '-']})
         self.assertEqual(spinner._interval, 321)
 
+    def test_invalid_placement(self):
+        """Test invalid placement of spinner.
+        """
+
+        with self.assertRaises(ValueError):
+            Halo(placement='')
+            Halo(placement='foo')
+            Halo(placement=None)
+
+        spinner = Halo(placement='left')
+        with self.assertRaises(ValueError):
+            spinner.placement = ''
+            spinner.placement = 'foo'
+            spinner.placement = None
+
+    def test_default_placement(self):
+        """Test default placement of spinner.
+        """
+
+        spinner = Halo()
+        self.assertEqual(spinner.placement, 'left')
+
+    def test_right_placement(self):
+        """Test right placement of spinner.
+        """
+        spinner = Halo(text='foo', placement='right', stream=self._stream)
+        spinner.start()
```

Comment:
I would suggest you add a `time.sleep` between `.start` and `.succeed` which will not stop the spinner immediately and there would be output that you can assert on. Perhaps this is the reason why test is failing on Appveyor. Get the output after `time.sleep` and succeeding.